SimCalorimetry/HGCalAssociatorProducers/plugins/LayerClusterAssociatorByEnergyScoreProducer.cc

```diff
@@ -42,9 +42,9 @@ void LayerClusterAssociatorByEnergyScoreProducer::produce(edm::StreamID,
                                                           edm::Event &iEvent,
                                                           const edm::EventSetup &es) const {
   rhtools_->getEventSetup(es);
-  edm::Handle<std::map<DetId, const HGCRecHit *>> hitMapHandle;
-  iEvent.getByToken(hitMapToken_, hitMapHandle);
-  const std::map<DetId, const HGCRecHit *> *hitMap = &*hitMapHandle;
+  edm::Handle<std::unordered_map<DetId, const HGCRecHit *>> hitMapHandle;
+  iEvent.getByToken(hitMap_, hitMapHandle);
+  const std::unordered_map<DetId, const HGCRecHit *> *hitMap = &*hitMapHandle;
```

Comment:
This could also be expressed as
```suggestion
  const std::unordered_map<DetId, const HGCRecHit *> *hitMap = &iEvent.get(hitMap_);
```